snippets/deployment.yaml

```diff
@@ -0,0 +1,22 @@
+name: deployment
+label: Create Kubernetes Deployment
```

Comment:
Consider either "Kubernetes Deployment" or "Create a Deployment manifest".

Reply:
I think `Kubernetes Deployment` is better, during a arbitrary  yaml editing, Kubernetes will give user a definitive  hint about this snippet.